packages/expo-file-system/src/next/FileSystem.types.ts

```diff
@@ -95,4 +103,14 @@ export declare class File {
    * Creates a file.
    */
   create(): void;
+
+  /**
+   * Copies a directory.
+   */
+  copy(to: Directory | File);
```

Comment:
Does it make sense to copy from `File` to `Directory`?
Also, `to` as an argument name in JS is quite weird. I would suggest something like `destination` or `target`.